### PR TITLE
GEOMESA-1043,GEOMESA-1042 Adding tracking and progress to tools inges…

### PR DESCRIPTION
**Pull request by elahrvivaz**
…t, redirecting stacktraces to logs in tools ingest

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
You now get lovely status bars in both local and ingest mode like so:
```
[=======================================================>    ]  95% 00:05:42 
```

File: geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/CountingInputStream.scala [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+/*
+ * Copyright (C) 2007 The Guava Authors
```
Comment (jahhulbert-ccri) [inline geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/CountingInputStream.scala]: nice. 2x licenses.

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ConverterIngest.scala [hunk 2]
```diff
@@ -100,26 +123,90 @@ class ConverterIngest(dsParams: Map[String, String],
...
+
+    val start = System.currentTimeMillis()
     val es = Executors.newFixedThreadPool(numLocalThreads)
     files.foreach(f => es.submit(new LocalIngestWorker(f)))
     es.shutdown()
-    es.awaitTermination(4, TimeUnit.DAYS)
 
-    logger.info(s"Local ingestion complete: ${getStatInfo(success.get, failure.get)}")
-  }
+    while (!es.isTerminated) {
```
Comment (jahhulbert-ccri) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ConverterIngest.scala]: hehehe yes I totally forgot to take out the 4 Day termination good catch :+1:

**Comment by elahrvivaz** [comment]
@jahhulbert-ccri think i should add running counters of succeeded/failed?

**Comment by jahhulbert-ccri** [comment]
e6ff42741fca97bb741f7a067f2fb5c873e6e054

**Pull request closed without merging**